system/mount.py

```diff
@@ -124,6 +126,8 @@ def set_mount(module, **kwargs):
     origname =  args['name']
     # replace any space in mount name with '\040' to make it fstab compatible (man fstab)
     args['name'] = args['name'].replace(' ', r'\040')
+    # remove trailing slash in mount name (if exists)
+    args['name'] = re.sub('/$','',args['name'])
```

Comment:
I'm no expert either, but I think you can just commit new changes and this
PR will be updated with them. That said, I don't know if the project wants
to handle it differently.  Maybe there is some info in the contribution
info?

On Sun, Nov 22, 2015, 12:51 zimo2001 <notifications@github.com> wrote:

> In system/mount.py
> <https://github.com/ansible/ansible-modules-core/pull/2464#discussion_r45557890>
> :
>
> > @@ -124,6 +126,8 @@ def set_mount(module, **kwargs):
> >      origname =  args['name']
> >      # replace any space in mount name with '\040' to make it fstab compatible (man fstab)
> >      args['name'] = args['name'].replace(' ', r'\040')
> > +    # remove trailing slash in mount name (if exists)
> > +    args['name'] = re.sub('/$','',args['name'])
>
> Hi @peerster <https://github.com/peerster>!
>
> Thanks for the feedback. You're absolutely right, rstrip is a better
> method.
>
> But - I'm kinda new at github...
> Should I edit the code or create a new commit and then a new pull request?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ansible/ansible-modules-core/pull/2464/files#r45557890>
> .
>